### DISCUSSION TITLE
Tooltip inside of overflow: hidden component

### DISCUSSION THREAD

**Discussion by Hurtak** [mantinedev/mantine#3718] [Q&A] [answered]
We are using Tooltip inside of `Card` component that has `overflow:hidden`. 

The problem is that since the Card's overflow hidden cuts off anything outside of its borders, the tooltip can only be positioned inside the Card and sometimes there is not enough space and it does not fit.
What is the best way to deal with this? Is there some way to make the Tooltip use portals and render independent of the containers that might have changed overflow? I tried to disable the overflow: hidden but we also have it on some other parent components, so it is not that straightforward.

**Comment by rtivital** [comment, marked as answer]
Set `<Tooltip withinPortal />`

**Reply by Hurtak** [reply]
Thank you! Now I feel stupid because I completely missed it even tho I went through the docs and the component props page.
Perhaps it could make sense to add this between the examples on the main docs page for Tooltip (https://mantine.dev/core/tooltip/)? I think this might be common enough use-case that it could be there.
